R/hiper_glm.R

```diff
@@ -1,6 +1,6 @@
 #' @export
```

Comment:
can't comment on the files themselves, but the .DS_Store files should be gitignored